### PR TITLE
Make the path changes to MessageProvider and HelloworldActivity

### PR DESCRIPTION
**Pull request by JAReddy**
Make the path changes to MessageProvider and HelloworldActivity from com.ddd to net.discdd

### CODE REVIEW

File: BundleClient/app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -23,7 +23,7 @@
         android:label="@string/app_name"
         android:theme="@style/Base.V7.Theme.AppCompat.Light" >
         <activity
-            android:name=".HelloworldActivity"
+            android:name="net.discdd.bundleclient.HelloworldActivity"
```
Comment (breed) [inline BundleClient/app/src/main/AndroidManifest.xml:26]: if you change the package on line 3 to net.discdd.bundleclient I don't think you need this change.

**Review by breed — CHANGES_REQUESTED** [review]
i think the packet="net.discdd.bundleclient" should also be fixed at the top.

File: BundleClient/app/src/main/AndroidManifest.xml (hunk 1 above)
Comment (breed) [reply BundleClient/app/src/main/AndroidManifest.xml:26]: i'm wrong, we are not supposed to use the package anymore.

Comment (JAReddy) [reply BundleClient/app/src/main/AndroidManifest.xml:25]: Removed it.

**Review by breed — APPROVED** [review]

**Pull request merged**